modules/openapi-generator/src/main/resources/JavaSpring/apiController.mustache

```diff
@@ -34,6 +34,9 @@ import javax.validation.Valid;
 {{#jdk8}}
 import java.util.Optional;
 {{/jdk8}}
+{{#useOptional}}
+import java.util.Optional;
+{{/useOptional}}
```

Comment:
If `jdk8` *and* `useOptional` are enabled, this will lead to a compile error because of the duplicate import.

Reply:
Thanks @joschi , updated accordingly.